lms/static/js/courseware/toggle_element_visibility.js

```diff
@@ -12,11 +12,14 @@
                     var show_text = actionElement.data('show');
                     var hide_text = actionElement.data('hide');
 
+                    actionElement.attr('aria-expanded', elementIsHidden);
+
                     if (elementIsHidden) {
                         if (hide_text) {
                             actionElement.html(actionElement.data('hide'));
                         } else {
                             actionElement.hide();
+                            $('.old-updates .toggle-visibility-button').first().focus();
```

Comment:
NIT: please make this a variable at the top of the file so it is clear what element we are taking actions on.